pages/1.11/installing/troubleshooting/index.md

```diff
@@ -61,11 +61,11 @@ When troubleshooting problems with a DC/OS installation, you should explore the
 Ensure that services that bind to port `53`, which is required by DNS Forwarder (`dcos-net.service`), are disabled and stopped. For example:
 
 
-   ```bash
-   sudo systemctl disable dnsmasq && sudo systemctl stop dnsmasq
-   ```
+    ```bash
```

Comment:
This appears misformatted on my local build (can see back ticks) 
![untitled](https://user-images.githubusercontent.com/18082037/42118780-ff90a022-7bba-11e8-8b66-3495256088d2.jpeg)